packages/fx-core/src/plugins/resource/bot/v2/plugin.ts

```diff
@@ -80,6 +80,8 @@ export class TeamsBotV2Impl {
     const bicepContext = {
       plugins: plugins.map((obj) => obj.name),
       configs: bicepConfigs,
+      enableAlwaysOn: isBotNotificationEnabled(),
```

Comment:
I think `isBotNotificationEnabled` is always true. Can we just remove this config and always set true in bicep?